PocketKit/Sources/PocketKit/PremiumUpgrade/Model/PocketSubscriptionStore.swift

```diff
@@ -39,13 +39,6 @@ final class PocketSubscriptionStore: SubscriptionStore, ObservableObject {
             }
             // Restore a purchased subscription, if any
             await self.fetchActiveSubscription()
-
```

Comment:
are we still sending receipt on launch?

Reply:
Yep, this method is called at `init()` time which happens on launch when all the other `Services` are initialized

Reply:
What it does:
1. it will try to fetch a subscription to the App Store; if it does, it will send a receipt with the subscription data
2. if it does not, it will send a receipt with no subscription data, of type "restore", so if one exists that was not retrieved, the backend would.